phoenicis-scripts/src/main/java/org/phoenicis/scripts/engine/implementation/PhoenicisScriptEngine.java

```diff
@@ -18,23 +18,25 @@
 
 package org.phoenicis.scripts.engine.implementation;
 
-import com.google.common.util.concurrent.Runnables;
-
-import java.io.InputStreamReader;
-import java.util.function.Consumer;
-
-public interface PhoenicisScriptEngine {
-    void eval(InputStreamReader inputStreamReader, Consumer<Exception> errorCallback);
-
-    void eval(String script, Runnable doneCallback, Consumer<Exception> errorCallback);
-
-    default void eval(String script, Consumer<Exception> errorCallback) {
-        eval(script, Runnables.doNothing(), errorCallback);
-    }
-
-    Object evalAndReturn(String line, Consumer<Exception> errorCallback);
-
-    void put(String name, Object object, Consumer<Exception> errorCallback);
-
-    void addErrorHandler(Consumer<Exception> errorHandler);
```

Comment:
Because I'm removing the `error handlers` contained in the script engine we need to provide them nearer to where they are needed in specific try/catch blocks. I think this is a better approach because it is currently very hard to know what error handlers are called for a script because they are often specified in an entirely different code region.